### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by snayrouz**
LIST OF CHANGES:
- Added styling to homepage
- Added styling to categories index
- Added styling to items index

THINGS I'M UNSURE ON:
I'm unsure of the people in the pictures. I wouldn't hang out with them. But it is what it is

RSPEC GREEN?

NO- WHY?

Because we changed 'Cart' to an icon of a shopping cart

*MENTION GROUP MEMBER TO REVIEW PR*

### CODE REVIEW

File: app/assets/stylesheets/custom.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.logout {
+  margin-right: 10px;
+}
+
+.leitbild-text{
```
Comment (snayrouz) [inline app/assets/stylesheets/custom.css:5]: had to add this custom CSS for the text to sit in the middle of the images

@sidewinder2020 just incase you want to change the picture descriptions, this is where you'll find that custom css and class. The class is leitbild-text on the index.html.erb of the home

File: app/views/categories/show.html.erb [hunk 2]
```diff
@@ -1,10 +1,23 @@
-<%= link_to "View Cart", cart_path %>
```
Comment (snayrouz) [inline app/views/categories/show.html.erb:1]: deleted this because we put the icon in

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <!-- <img class= src="/images/g" alt="First slide"> -->
+        <%= image_tag("basket_2.jpg", class: "d-block w-80 img-fluid") %>
+        <div class="carousel-caption d-none d-md-block leitbild-text">
+          <h3>Baskets</h3>
+        <p>Here are all the baskets!</p>
+        </div>
+      </div>
+      <div class="carousel-item">
```
Comment (snayrouz) [inline app/views/home/index.html.erb:12]: if anyone wants to change the pictures go for it, just make sure it put it in the assets/images folder.

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -9,14 +9,46 @@
...
+          <a class="dropdown-item" href="/basket">Basket</a>
+          <a class="dropdown-item" href="/basketmaking">Basket Making Supplies</a>
+          <a class="dropdown-item" href="/scuba">Scuba Gear</a>
+          <a class="dropdown-item" href="/snorkel">Snorkel Gear</a>
+        </div>
+      </li>
+        </ul>
+
+        <%= link_to image_tag("glyphicons-203-shopping-cart.png"), cart_path, class: "logout" %>
+        <% if logged_in? %>
```
Comment (snayrouz) [inline app/views/layouts/application.html.erb:39]: this toggles the login/logout functionality

**Comment by snayrouz** [comment]
@blsrofe @Boots2291 @sidewinder2020 

can someone merge this before it gets too far away from master

**Comment by ButeraMV** [comment]
Taking a look at the conflicts

**Comment by ButeraMV** [comment]
@snayrouz Let me know if that conflict resolution looks good. I think all I took out was the old HTML.

File: app/views/items/index.html.erb [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
-      </thead>
-<tbody>
-  <tr><% @items.each do |item| %></tr>
-    <td><%= item.title %></td>
-    <td><%= item.description %></td>
-    <td><%= item.price %></td>
-    <td><%= link_to image_tag(item.image), item_path(item) %></td>
-  <% end %>
-</tbody>
+    <thead>
```
Comment (snayrouz) [inline app/views/items/index.html.erb]: @Boots2291 ya you're right it was the old html.

Cool I'll merge it and then master will be up to date

**Pull request merged**